### ISSUE TITLE
accessibility issue 22 (medium) - modal close button contrast

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#242]
The “X” or close button on the modal dialog has insufficient color contrast.
The expected color contrast ratio for interface components  is at least 3:1.

**Comment by yulgit1** [comment]
resolved: https://github.com/ycba-cia/blacklight-collections2/commit/2ccd07e77391ba1f36f455df655c907db3a02ca2
